### PR TITLE
Fix some ir printer bugs

### PR DESCRIPTION
**Pull request by bertmaher**

### CODE REVIEW

File: torch/csrc/jit/tensorexpr/ir_printer.cpp [hunk 1]
```diff
@@ -231,12 +231,12 @@ void IRPrinter::visit(const Cond* v) {
     os() << false_stmt << std::endl;
     os() << "}";
   } else {
-    os() << "if(" << cond << ") {" << std::endl;
+    os() << "if(" << *cond << ") {" << std::endl;
     os() << true_stmt << std::endl;
```
Comment (ZolotukhinM) [inline torch/csrc/jit/tensorexpr/ir_printer.cpp]: We need to dereference `true_stmt` too

**Review by ZolotukhinM — APPROVED** [review]
Otherwise looks good!

File: torch/csrc/jit/tensorexpr/ir_printer.cpp [hunk 2]
```diff
@@ -231,12 +231,12 @@ void IRPrinter::visit(const Cond* v) {
     os() << false_stmt << std::endl;
```
Comment (ZolotukhinM) [inline torch/csrc/jit/tensorexpr/ir_printer.cpp]: And here as well :)

Comment (bertmaher) [reply torch/csrc/jit/tensorexpr/ir_printer.cpp]: thanks!  I think I got 'em all this time :)

**Review by lly-zero-one — APPROVED** [review]

**Pull request merged**